evolver/calibration/standard/calibrators/temperature.py

```diff
@@ -19,14 +19,26 @@ class TemperatureCalibrator(IndependentVialBasedCalibrator):
 
     class CalibrationData(Transformer.Config):
         measured: Dict[int, Dict[str, List[float]]] = {}  # {vial_index: {"reference": [], "raw": []}}
+        completed_actions: List[str] = []
```

Comment:
Since `completed_actions` looks like something we want to support at the interface, perhaps this should actually go in `Transformer.Config` (or actually I think better yet: this class should be based off `Calibrator.CalibrationData` and that should have `completed_actions`)

Reply:
makes sense, will fix.